### DISCUSSION TITLE
just thanking!

### DISCUSSION THREAD

**Discussion by HendrikPetertje** [nvimdev/lspsaga.nvim#708] [General]
I ran a PackerSync this morning to see a lot of fixes and quality of live improvements to the huge patch you did a few days ago. 
Thank you so much for working on this project and making my Nvim LSP experience (and many others) look like Unicorns & Rainbows!

enjoy the 🍺 or ☕ coming your way!

**Comment by glepnir** [comment]
because you give me power:) I had add you into the backers
